### PR TITLE
Adding copyright headers

### PR DESCRIPTION
**Pull request by truongnh1992**
Signed-off-by: Nguyen Hai Truong <nguyenhaitruonghp@gmail.com>

### CODE REVIEW

**Comment by longkb** [comment]
LGTM :)

File: go/bucketpool/bucketpool.go [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2018 Google Inc.
```
Comment (deepthi) [inline go/bucketpool/bucketpool.go]:
```suggestion
Copyright 2019 The Vitess Authors
```

File: go/bucketpool/bucketpool_test.go [hunk 2]
```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2018 Google Inc.
```
Comment (deepthi) [inline go/bucketpool/bucketpool_test.go]:
```suggestion
Copyright 2019 The Vitess Authors
```

**Comment by truongnh1992** [comment]
@deepthi I've just resolved your request changes. Thanks!

**Pull request merged**